presto-main/src/test/java/io/prestosql/server/TestDynamicFilterService.java

```diff
@@ -434,6 +435,75 @@ public void testReplicatedDynamicFilter()
         assertEquals(dynamicFiltersStageSupplier.getRequestCount(), 1);
     }
 
+    @Test
+    public void testDynamicFilterCancellation()
+    {
+        DynamicFilterService dynamicFilterService = new DynamicFilterService(new FeaturesConfig());
+        DynamicFilterId filterId = new DynamicFilterId("df");
+        Expression df1 = expression("DF_SYMBOL1");
+        QueryId queryId = new QueryId("query");
+        StageId stageId = new StageId(queryId, 0);
+        List<TaskId> taskIds = ImmutableList.of(new TaskId(stageId, 0), new TaskId(stageId, 1));
+
+        TestDynamicFiltersStageSupplier dynamicFiltersStageSupplier = new TestDynamicFiltersStageSupplier(RUNNING);
+        dynamicFiltersStageSupplier.addTasks(taskIds);
+        dynamicFilterService.registerQuery(queryId, dynamicFiltersStageSupplier, ImmutableSet.of(filterId), ImmutableSet.of(filterId), ImmutableSet.of());
+        ColumnHandle column = new TestingColumnHandle("probeColumnA");
+        DynamicFilter dynamicFilter = dynamicFilterService.createDynamicFilter(
+                queryId,
+                ImmutableList.of(new DynamicFilters.Descriptor(filterId, df1)),
+                ImmutableMap.of(
+                        Symbol.from(df1), column));
+        assertFalse(dynamicFilter.isBlocked().isDone());
+        assertFalse(dynamicFilter.isComplete());
+        assertEquals(dynamicFilter.getCurrentPredicate(), TupleDomain.all());
+
+        // assert initial dynamic filtering stats
```

Comment:
remove, we don't need to assert these in every test

Reply:
removed